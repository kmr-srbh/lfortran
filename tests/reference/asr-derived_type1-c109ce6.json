{
    "basename": "asr-derived_type1-c109ce6",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/derived_type1.f90",
    "infile_hash": "ed39e2004e29457f0f83e6a1badf6ecbee98749ca624594fc0160e52",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-derived_type1-c109ce6.stdout",
    "stdout_hash": "573b65b0b85a691b60af9344815a83e1a515194bce501827d826dfe9",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}