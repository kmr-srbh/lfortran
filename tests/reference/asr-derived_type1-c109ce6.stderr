semantic error: The RHS character len must be known at compile time
  --> tests/derived_type1.f90:11:5 - 12:33
   |
11 |        character(kind=tfc, len=*), public, parameter :: nl = &
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^...
...
   |
12 |            &' '//toml_escape%newline
   | ...^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
